src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx

```diff
@@ -379,49 +348,17 @@ describe('SelectMultiple', () => {
     expect(screen.getByRole('option', { name: /a selected/ })).toBeVisible();
   });
 
-  test('search with select and clear', async () => {
-    const user = userEvent.setup();
-    const defaultOptions = [
-      'Apple',
-      'Orange',
-      'Banana',
-      'Grape',
-      'Melon',
-      'Strawberry',
-      'Kiwi',
-      'Mango',
-      'Raspberry',
-      'Rhubarb',
-    ];
-    const Test = () => {
-      const [options, setOptions] = useState(defaultOptions);
-      const [valueMultiple, setValueMultiple] = useState([]);
-      return (
-        <Grommet>
-          <SelectMultiple
-            options={options}
-            value={valueMultiple}
-            placeholder="Select"
-            onSearch={(text) => {
-              const escapedText = text.replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
-              const exp = new RegExp(escapedText, 'i');
-              setOptions(defaultOptions.filter((o) => exp.test(o)));
-            }}
-            onClose={() => setOptions(defaultOptions)}
-            onChange={({ value }) => {
-              setValueMultiple(value);
-            }}
-          />
-        </Grommet>
-      );
-    };
-    render(<Test />);
+  test('search with select and clear', () => {
+    render(<TestManyOptions />);
```

Comment:
Could you elaborate on how pulling this out helps with timing? I've seen the pattern before and in support, but didn't realize it affect timing.

Reply:
Yeah this was just to make the readability of the tests a little cleaner, it doesn't affect timing